discovery-provider/src/queries/get_health_test.py

```diff
@@ -282,7 +283,7 @@ def getBlock(_u1, _u2):  # unused
     assert health_results['total_memory'] == 6237151232
     assert health_results['used_memory'] == 3055149056
     assert health_results['transferred_bytes_per_sec'] == 7340.780857447676
-
+    assert health_results['number_of_cpus'] == os.cpu_count()
```

Comment:
FYI chatted with Vicky about this - IMO this is preferable to globally mocking os.cpu_count() or adding a cpu_count var in the config and mocking config (b/c ofc this isn't really config).